index.js

```diff
@@ -123,11 +111,12 @@ const Home = () => {
   // 3. Fetch the /?meta for the requested package
   react.useEffect(() => {
     // Reset any previous state
-    if (!request.package) {
-      setFile({});
-      setFetchError(false);
+    if (!state.request.package) {
+      dispatch({ type: 'setFile', payload: {} });
+      dispatch({ type: 'setFetchError', payload: true });
```

Comment:
Should the payload here not be `false`?

At the moment, the homepage causes a `404` on this branch

Reply:
Ack great, thanks for finding this!